clodius/tiles/beddb.py

```diff
@@ -191,6 +207,9 @@ def list_items(db_file, start, end, max_entries=None):
     max_entries: int
         The maximum number of results to return
     """
+    ts_info = tileset_info(db_file)
+    version = ts_info["version"]
+    print("version:", version)
```

Comment:
Is this print statement suppose to be there?

Reply:
Certainly not 😠